samples/sample_multi_transcode/src/pipeline_transcode.cpp

```diff
@@ -2573,6 +2585,38 @@ MFX_IOPATTERN_IN_VIDEO_MEMORY : MFX_IOPATTERN_IN_SYSTEM_MEMORY);
 #endif
     }
 
+    if (pInParams->nAvcTemp)
+    {
+        if (pInParams->EncodeId == MFX_CODEC_HEVC)
+        {
+            int i;
+            for (i = 0; i < 8; i++)
+            {
+                m_AvcTemporalLayers.Layer[i].Scale = pInParams->nAvcTemporalLayers[i];
+                m_AvcTemporalLayers.BaseLayerPID = pInParams->nBaseLayerPID;
```

Comment:
why this is in the loop?

Reply:
Fixed. Thanks.
Updated commit:  3f5bc4f